src/projects/shared/api/circuit/tests/public/Component.test.ts

```diff
@@ -11,66 +11,295 @@ import {CreateTestRootCircuit} from "tests/helpers/CreateTestCircuit";
 
 describe("Component", () => {
     describe("Bounds", () => {
-        // Test ideas:
-        // Bounds for the TestComp are 1x1 I think
+        test("1x1 default", () => {
+            const [circuit, _, {PlaceAt}] = CreateTestRootCircuit();
+            const [c] = PlaceAt(V(0, 0));
+            expect(c.bounds.center).toStrictEqual(V(0, 0));
+            expect(c.bounds.size).toStrictEqual(V(1, 1));
+        });
+
+        test("1x1 with rotation", () => {
+            const [circuit, _, {PlaceAt}] = CreateTestRootCircuit();
+            const [c] = PlaceAt(V(0, 0));
+            c.angle = Math.PI / 2;
+            expect(c.bounds.center).toStrictEqual(V(0, 0));
+            expect(c.bounds.size).toStrictEqual(V(1, 1));
+        });
+
+        test("1x1 default offset start", () => {
+            const [circuit, _, {PlaceAt}] = CreateTestRootCircuit();
+            const [c] = PlaceAt(V(-1, -1));
+            expect(c.bounds.center).toStrictEqual(V(-1, -1));
+            expect(c.bounds.size).toStrictEqual(V(1, 1));
```

Comment:
Might be clearer to just do `expect(c.bounds).toEqual(Rect.from({ cx: -1, cy: -1, width: 1, height: 1 }))` (and for above)